ios/MattermostShare/ShareViewController.swift

```diff
@@ -44,10 +44,16 @@ class ShareViewController: SLComposeServiceViewController {
   override func isContentValid() -> Bool {
     // Do validation of contentText and/or NSExtensionContext attachments here
     if (attachments.count > 0) {
+      let maxImageSize = store.getMaxImageSize()
```

Comment:
I still fail to see the difference between this and MaxFileSize, and if this only applies to Images then we need to inspect the attachment type before running this check